htlcswitch/link.go

```diff
@@ -2344,6 +2344,9 @@ func (l *channelLink) canSendHtlc(policy ForwardingPolicy,
 
 	// Check to see if there is enough balance in this channel.
 	if amt > l.Bandwidth() {
+		l.log.Errorf("insufficient bandwidth to route htlc on "+
```

Comment:
I'd say this should be a `Warnf` or perhaps even `Infof`.

Reply:
Me, too. What about the other errors in the file showing (very) similar messages? I think most of them are not lnd errors, but instead rather usual reasons for failed payments.

Reply:
Yeah, agree, the log levels could be lowered for some of these.

Reply:
I'd be happy to do that. I think every message that is caused by the incoming transaction conflicting with public/non-public channel state should have be a warning instead.